config/checkstyle-examples-checks.xml

```diff
@@ -103,6 +103,11 @@
     <property name="eachLine" value="false"/>
   </module>
 
+  <module name="RegexpSingleline">
+    <property name="format" value="Files\.createTempDirectory"/>
```

Comment:
Please add it at https://github.com/checkstyle/checkstyle/blob/9554a5367c348c79f3abed3557537f508bec8831/config/signatures.txt#L5

It is better way to forbid some method

Reply:
````
createTempFile(Path dir,
                                      String prefix,
                                      String suffix,
                                      FileAttribute<?>... attrs)
````
````
createTempFile(String prefix,
                                      String suffix,
                                      FileAttribute<?>... attrs)
````

Reply:
@rnveach, @romani i have made the changes to my best understanding. please let me know whenever convenient for you if changes are right.

Reply:
I am continuting to get failures in my local. An existing test method in PropertyCacheFileTest.java is using the below method signature. 

```
public static File createTempFile(String prefix, String suffix,
                                      File directory)

```

Reference method using above signature:
```
 @Test
    public void testInCache() throws IOException {
        final Configuration config = new DefaultConfiguration("myName");
        final String filePath = File.createTempFile("junit", null, temporaryFolder).getPath();
        ...
        }

```

And, despite trying to use it in the signature-test.txt, I am still getting parsing failures. 

`com.puppycrawl.tools.checkstyle.PropertyCacheFileTest#createTempFile(java.lang.String, java.lang.String, java.io.File) @ Use of this method is forbidden, please use @TempDir for better temporary directory control`

Reply:
Error:
`Failed to execute goal de.thetaphi:forbiddenapis:3.6:testCheck (forbiddenapis-test) on project checkstyle: Parsing signatures failed: Method not found while parsing signature: com.puppycrawl.tools.checkstyle.PropertyCacheFileTest#createTempFile(java.lang.String, java.lang.String, java.io.File)`